analysis/rogueassassination/src/modules/shadowlands/conduits/LethalPoisons.tsx

```diff
@@ -0,0 +1,67 @@
+import SPELLS from 'common/SPELLS';
+import Analyzer, { Options } from 'parser/core/Analyzer';
+import calculateEffectiveDamage from 'parser/core/calculateEffectiveDamage';
+import { SELECTED_PLAYER } from 'parser/core/EventFilter';
+import Events, { DamageEvent } from 'parser/core/Events';
+import ConduitSpellText from 'parser/ui/ConduitSpellText';
+import ItemDamageDone from 'parser/ui/ItemDamageDone';
+import Statistic from 'parser/ui/Statistic';
+import STATISTIC_CATEGORY from 'parser/ui/STATISTIC_CATEGORY';
+
+const DAMAGE_BONUS = [
+  0,
+  0.1,
+  0.11,
+  0.12,
+  0.13,
+  0.14,
+  0.15,
+  0.16,
+  0.17,
+  0.18,
+  0.19,
+  0.2,
+  0.21,
+  0.22,
+  0.23,
+  0.24,
+];
+
+class LethalPoisons extends Analyzer {
+  conduitRank: number = 0;
+  bonusDamage: number = 0;
+
+  constructor(options: Options) {
+    super(options);
+    this.active = this.selectedCombatant.hasConduitBySpellID(SPELLS.LETHAL_POISONS.id);
+    this.conduitRank = this.selectedCombatant.conduitRankBySpellID(SPELLS.LETHAL_POISONS.id);
+
+    this.addEventListener(
+      Events.damage
+        .by(SELECTED_PLAYER)
+        .spell([
+          SPELLS.DEADLY_POISON,
+          SPELLS.DEADLY_POISON_DOT,
+          SPELLS.DEADLY_POISON_PROC,
+          SPELLS.WOUND_POISON,
+        ]),
+      this.onDamage,
+    );
+  }
+
+  onDamage(event: DamageEvent) {
+    this.bonusDamage += calculateEffectiveDamage(event, DAMAGE_BONUS[this.conduitRank]);
+  }
```

Comment:
not something that I want you to add to this PR (it is already big enough :P), but this pattern is so common that we should probably just have a `DamageBonusAnalyzer` for it or something. You've got 3-4 conduit implementations that are just this with the numbers and spells changed.

Reply:
I'll look into implementing something that here soon to cut down on this duplicative code.